examples/sample-app/README.md

```diff
@@ -163,26 +169,17 @@ This section covers how to perform all the steps of building, deploying, and upd
     be added to the docker-registry service list so that it's reachable from other places.
 
 
-6. Confirm the registry is accessible (you may need to run this more than once):
-
-        $ curl -H 'Accept: application/json' -v `oc get service docker-registry --template="{{ .spec.portalIP }}:{{ with index .spec.ports 0 }}{{ .port }}{{ end }}/v2/" --config=openshift.local.config/master/admin.kubeconfig`
-
-    You should see:
-
-        "< Docker-Distribution-Api-Version: registry/2.0"
-
-
-7. Bind a user names `test-admin` to the `view` role in the default namespace so you can observe progress in the web console
+6. Bind a user names `test-admin` to the `view` role in the default namespace so you can observe progress in the web console
 
         $ oadm policy add-role-to-user view test-admin --config=openshift.local.config/master/admin.kubeconfig
 
 
-9. Login as `test-admin` using any password
+7. Login as `test-admin` using any password
 
         $ oc login --certificate-authority=openshift.local.config/master/ca.crt
 
 
-9. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8443/console`.  Login using the user `test-admin` and any password.
+8. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8443/console`.  Login using the user `test-admin` and any password.
```

Comment:
Delete this step.  somehow we ended up with two of them?   (this one and step 10).  Anyway it makes more sense to have them login to the console after creating the project in step 9. that way they will actually be able to see something when they login.